lib/screens/projects_screen.dart

```diff
@@ -47,6 +51,7 @@ class _ProjectsScreenState extends State<ProjectsScreen> {
 
   @override
   Widget build(BuildContext context) {
+    loadProjects();
```

Comment:
This loadProjects function sets the state. This could lead to an infinite loop. It is better to move this function out of the build method.

Reply:
Thanks have done it via Navigator.push().then((var)->loadProject())